src/Services/PatientService.php

```diff
@@ -226,7 +226,7 @@ public function databaseUpdate($data)
         $sqlResult = sqlStatement($sql, $query['bind']);
 
         if (
-            $dataBeforeUpdate['care_team_provider'] != $data['care_team_provider']
+            $dataBeforeUpdate['care_team_provider'] != ($data['care_team_provider'] ?? '')
```

Comment:
noticed that we're unable to remove the `care_team_*` values from `patient_data` in `Choices` if want to un-assign and the care_team_facility has a select 2 issue displaying the full width

Reply:
Hmm.. I'm wondering if this will create an issue with the FHIRCareTeamService?  I'm guessing no, but my fried brain is wondering if this could be a problem.  Could just be the fried brain though...

Reply:
for clarity, unassign means setting to `none`

Reply:
think this is ready to bring in @bradymiller if you don't mind reviewing and then can start an issue for the 2 things noted here before I have all my php8 fixins cut off at the pass by you :)